docs/admin/installation/hardware.rst

```diff
@@ -733,11 +683,9 @@ For the hardware we recommend, you can find a list of end-of-life dates below:
 ===================  ====================================================
 Hardware             End-of-Life (EOL)                                       
 ===================  ====================================================
-Intel NUC12WSKi5     Not yet confirmed
```

Comment:
I'd leave the EOL dates in for the older hardware for informational purposes.

Reply:
OK, done.